css/templates/gallery/page.css

```diff
@@ -21,9 +21,85 @@
 @import '../../core.css';
 
 @import './page-vars';
+@import './components/_landing-images';
 
 body {
   color: var(--font-color-primary);
   font-family: var(--font-family-primary);
   line-height: var(--line-height);
 }
+
+.gallery-base-content {
+  background: #fff;
```

Comment:
i think some basic colors are defined as variables (white and black), prefer to use the variables.